### PR TITLE
feat: get likes/followers/followings

### LINKED ISSUE #16: Display likers: server
Add endpoint to get list of people who liked a post

### PR DESCRIPTION
**Pull request by r1oga**
Closes #16 #17 #18 
Add new handlers on following routes:

|handler|route|description|
|---|---|---|
|`handleGetLikesByPost`|`/v1/post/:hash/likes`|returns list of users who liked a post|
|`handleGetUserFollowers`|`/v1/:user/followers`|returns list of followers a given user has|
|`handleGetUserFollowings`|`/v1/:user/followings`|return list of users of given user follows|

### Test
- unit tests: `npm run test:some http`
- example of manual requests:
  - likes:
    ```
    curl -X GET --location "http://localhost:3000/v1/post/0xC7E20724c409d9E8e3181a559F4c529B17dc1Bd7%2Fb298e95069867db8d2f5152d50f91029fbba01225f557bc180895f450d47bc6c/likes"
    ```
  - followers:
    ```
    curl -X GET --location "http://localhost:3000/v1/0xd44a82dD160217d46D754a03C8f841edF06EBE3c/followers"
    ```
  
  - followings:
    ```
    curl -X GET --location "http://localhost:3000/v1/0xC7E20724c409d9E8e3181a559F4c529B17dc1Bd7/followings" 
    ```

### CODE REVIEW

File: src/models/moderations.ts [hunk 1]
```diff
@@ -132,6 +132,23 @@ const moderations = (sequelize: Sequelize) => {
...
+  ): Promise<string[]> => {
+    const result = await model.findAll({
+      attributes: ['creator'],
+      where: { reference, subtype: 'LIKE' },
+      offset,
+      limit,
+      order: [['createdAt', order]],
+    });
+
+    return result.map((r: any) => r.toJSON().creator);
```
Comment (r1oga) [inline src/models/moderations.ts:149]: creator vs messageId?
see https://github.com/zkitter/zkitterd/pull/21#discussion_r1000470190

File: src/models/connections.ts [hunk 2]
```diff
@@ -94,6 +94,40 @@ const connections = (sequelize: Sequelize) => {
...
+  ): Promise<ConnectionModel[]> => {
+    let result = await model.findAll({
+      attributes: ['name'],
+      where: { creator },
+      offset,
+      limit,
+      order: [['createdAt', order]],
+    });
+
+    return result.map((r: any) => r.toJSON().name);
```
Comment (r1oga) [inline src/models/connections.ts:128]: name vs messageId?
see https://github.com/zkitter/zkitterd/pull/21#discussion_r1000470190

File: src/models/connections.ts [hunk 3]
```diff
@@ -94,6 +94,40 @@ const connections = (sequelize: Sequelize) => {
...
+  ): Promise<ConnectionModel[]> => {
+    let result = await model.findAll({
+      attributes: ['creator'],
+      where: { name },
+      offset,
+      limit,
+      order: [['createdAt', order]],
+    });
+
+    return result.map((r: any) => r.toJSON().creator);
```
Comment (r1oga) [inline src/models/connections.ts:111]: creator vs messageId?
see https://github.com/zkitter/zkitterd/pull/21#discussion_r1000470190

Comment (0xtsukino) [reply src/models/connections.ts:111]: nvm let's keep as is

File: src/models/connections.ts (hunk 2 above)
Comment (0xtsukino) [reply src/models/connections.ts:128]: nvm let's keep as is

**Pull request merged**